app/controllers/account_activations_controller.rb

```diff
@@ -1,7 +1,7 @@
 class AccountActivationsController < ApplicationController
   def edit
     user = User.find_by(email: params[:email])
-    if user&.(!user.activated? && user.authenticated?(:activation, params[:id]))
+    if user && !user.activated? && user.authenticated?(:activation, params[:id])
```

Comment:
You can simplify this condition like this: 
unless user&.activated? && user&.authenticated?(:activation, params[:id])